chapters/3-measurement.tex

```diff
@@ -37,125 +39,209 @@
+and it is a major source of error.
+
+The data linkage table should indicate whether 
+datasets should be merge one-to-one, for example, 
+merging baseline data and endline data that use the same unit of observation,
 or whether two datasets should be merged many-to-one, 
 for example, school administrative data merged with student data.
-Your data map must indicate which ID variables can be used and how to merge datasets.
-It is common that administrative data use IDs 
-that are different than the project IDs, 
-and the linkage between those should be clear from your master dataset.
-
-The data map should also include metadata about the handling of all information.
-These characteristics may be updated as the project progresses.
-For example, you will need to note the original source of each dataset,
-as well as the project folder where 
-the raw original data and codebooks are stored
-and where the back-ups for the each raw dataset are stored.
-
-Some of the characteristics in your master datasets and your data map
-should be filled in during the planning stage,
-but both of them should be active resources 
-that are updated as your project evolves.
-Finally, your master data should not include any unlabeled missing values. 
-If the information is missing for one unit, 
-then the reason should always be indicated with a code.
-An example for such reason could be that a unit was not included in the treatment assignment
-as it was not sampled in the first place,
-or was not located in the data collection at a given round.
-
-\subsection{Defining study comparisons using master data}
-
-Your research design will determine what statistical comparisons you need
-to estimate in your analytical work.
+Your data map must indicate which ID variables 
+can be used -- and how -- when merging datasets.
+The data linkage table is also a great place to list other types of metadata,
+such as the source of your data, its backup locations, and so on.
+
+Second, you should then create one \textbf{master dataset} 
+for each unit of observation
+used in any significant research activity.
+Examples of such activities are data collection, data analysis, 
+sampling, and treatment assignment.
+The master dataset should include and be the authoritative source of
+all \textbf{research variables}\sidenote{
+	\textbf{Research variables:} Research data that identifies observations
+	and maps research design information to those observations.
+	Research variables are time-invariant and 
+	often, but not always, controlled by the research team.
+	Examples include
+	ID variables, sampling status, treatment status, and treatment uptake.} 
+but not include any \textbf{measurement variables}\sidenote{
+	\textbf{Measurement variables:} Data that 
+	corresponds to direct observations of the real world, 
+	recorded sentiments of the subjects of the research 
+	or any other aspect your project is studying. 
+	Research variables are not controlled by the research team
```

Comment:
shouldn't this say measurement variables?

Reply:
```suggestion
	Measurement variables are not controlled by the research team
```